rest/model/patch.go

```diff
@@ -155,24 +156,29 @@ func (apiPatch *APIPatch) BuildFromService(p patch.Patch, args *APIPatchArgs) er
 	apiPatch.buildBasePatch(p)
 
 	projectIdentifier := p.Project
-	proj, err := model.FindMergedProjectRef(projectIdentifier, p.Version, false)
-	if err != nil {
-		return errors.Wrapf(err, "finding project ref '%s'", projectIdentifier)
-	}
-	if proj == nil {
-		return errors.Errorf("project ref '%s' not found", projectIdentifier)
-	}
-	// Projects that use the GitHub merge queue cannot enqueue to the commit queue.
-	apiPatch.CanEnqueueToCommitQueue = (p.HasValidGitInfo() || p.IsGithubPRPatch()) && proj.CommitQueue.MergeQueue != model.MergeQueueGitHub
```

Comment:
This field is only used by the UI, right? Instead of including this here, could we instead remove CanEnqueueToCommitQueue from the APIPatch struct and instead just add a resolver for it? https://github.com/evergreen-ci/evergreen/blob/e0023201027c8ad77ac5d111fc2af09d2eca471f/graphql/version_resolver.go#L483

Reply:
(@khelif96 is that reasonable?)

Reply:
Yeah this makes sense to me